docs_user/modules/proc_adopting-the-orchestration-service.adoc

```diff
@@ -166,10 +159,10 @@ heatcfnapi.heat.openstack.org/heat-cfnapi   True     Setup complete
   Updated At: '2023-10-11T21:48:01.000000'
 ----
 
-. Verify you can now see your the {orchestration} stacks again. Test whether you can create networks, subnets, ports, or routers:
+. Verify that you can see your {orchestration} stacks. Test whether you can create networks, subnets, ports, or routers:
 +
```

Comment:
How would users test that they can create networks, subnets, ports, etc.? 
Are we only suggesting that they test these things, or is it mandatory?

Reply:
It was more a suggestion, I didn't plan on providing some example Heat template for the to actually do it.

I see what you're saying though, maybe it's best we drop that and just leave the first sentence there.